perception/traffic_light_visualization/src/traffic_light_roi_visualizer/shape_draw.cpp

```diff
@@ -0,0 +1,124 @@
+}
+
+void drawRightArrow(const DrawFunctionParams & params)
+{
+  int y_offset = params.size / 2 + 10;
+  // Start from the left, go to the right (start x < end x)
+  cv::arrowedLine(
+    params.image, cv::Point(params.position.x, params.position.y - y_offset),  // Start point
+    cv::Point(
+      params.position.x + params.size, params.position.y - y_offset),  // End point to the right
+    params.color, 2, 8, 0, 0.3);
+}
+
+void drawStraightArrow(const DrawFunctionParams & params)
+{
+  int y_offset = params.size / 2 + 10;  // This adjusts the base position upwards
+  // Start point is lower, and it goes upwards to a higher (smaller y-value) point
+  cv::arrowedLine(
+    params.image, cv::Point(params.position.x, params.position.y - y_offset),  // Start point
+    cv::Point(params.position.x, params.position.y - params.size - y_offset),  // End point
+    params.color, 2, 8, 0, 0.3);
+}
+void drawDownArrow(const DrawFunctionParams & params)
+{
+  int y_offset = params.size / 2 + 10;  // This adjusts the base position upwards
+  // Start point is higher, and it goes downwards to a lower (larger y-value) point
+  cv::arrowedLine(
+    params.image, cv::Point(params.position.x, params.position.y - y_offset),  // Start point
+    cv::Point(params.position.x, params.position.y + params.size - y_offset),  // End point
+    params.color, 2, 8, 0, 0.3);
+}
+
+void drawDownLeftArrow(const DrawFunctionParams & params)
+{
+  int y_offset = params.size / 2 + 15;
+  // Down-left arrow
+  cv::arrowedLine(
+    params.image, cv::Point(params.position.x, params.position.y - y_offset),
+    cv::Point(params.position.x - params.size, params.position.y + params.size - y_offset),
+    params.color, 2, 8, 0, 0.3);
+}
+
+void drawDownRightArrow(const DrawFunctionParams & params)
+{
+  int y_offset = params.size / 2 + 15;
+  // Down-right arrow
+  cv::arrowedLine(
+    params.image, cv::Point(params.position.x, params.position.y - y_offset),
+    cv::Point(params.position.x + params.size, params.position.y + params.size - y_offset),
+    params.color, 2, 8, 0, 0.3);
+}
+
+void drawCross(const DrawFunctionParams & params)
+{
+  int y_offset = params.size / 2 + 10;
+  // Cross (two lines intersecting)
+  int half_size = params.size / 2;
+  cv::line(
+    params.image,
+    cv::Point(params.position.x - half_size, params.position.y - half_size - y_offset),
+    cv::Point(params.position.x + half_size, params.position.y + half_size - y_offset),
+    params.color, 2);
+  cv::line(
+    params.image,
+    cv::Point(params.position.x + half_size, params.position.y - half_size - y_offset),
+    cv::Point(params.position.x - half_size, params.position.y + half_size - y_offset),
+    params.color, 2);
+}
+
+void drawTrafficLightShape(
+  cv::Mat & image, const std::string & shape, const cv::Point & position, const cv::Scalar & color,
+  int size)
+{
+  static std::map<std::string, DrawFunction> shapeToFunction = {
+    {"circle", drawCircle},
```

Comment:
I will suggest unicode replacements for these, don't merge now 🙏

Reply:
Forgot to reply to this, but i tried using unicode symbol
![Screenshot from 2024-05-12 01-00-48](https://github.com/autowarefoundation/autoware.universe/assets/36904941/49b5b52c-59d3-4d3b-828c-d1f9c92524b3)

it would just return "?" for the symbols i put, and when i tried to use the unicode hexcode for them it just printed out the actual hexcode instead of the symbol so i went with the cv drawing method instead